install-steps.md

```diff
@@ -488,15 +488,15 @@ Two approaches:
     export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}' | xargs)
```

Comment:
No need for grep or xargs:
```
curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/$VERSION/release.txt | awk '/Pull From: quay.io/ { print $3 }'
```

Reply:
see above

Reply:
I'm going to say this is doesn't matter and leaving the expression doesn't hurt considering we have validated with it.